util/schema/registry_schema.json

```diff
@@ -96,9 +96,33 @@
       "suggest": false
     },
     "domain": {
-      "level": "warning",
-      "description": "'domain' is a recommended field. The value is the scientific topic area addressed by the ontology.",
-      "type": "string"
+      "level": "error",
+      "description": "'domain' is a mandatory field. The value is the scientific topic area addressed by the ontology.",
+      "type": "string",
+      "enum": [
+        "upper",
+        "anatomy and development",
+        "health",
+        "phenotype",
+        "experiments",
+        "environment",
+        "biochemistry",
+        "microbiology",
+        "agriculture",
+        "diet, metabolomics and nutrition",
```

Comment:
would highly prefer this having an Oxford comma to reduce confusion later

Reply:
If you can be bothered to make that fix, I think an oxford comma is appropriate.

Reply:
```suggestion
        "diet, metabolomics, and nutrition",
```

Reply:
i added suggestions everywhere else too